client/cz-clock-dash.html

```diff
@@ -0,0 +1,108 @@
+<link rel="import" href="../bower_components/polymer/polymer.html">
+<link rel="import" href="../bower_components/paper-card/paper-card.html">
+<link rel="import" href="../bower_components/paper-item/paper-item.html">
+
+<dom-module id="cz-clock-dash">
+
+  <template>
+    <style>
+      paper-card .card-flex {
+        display: flex;
+        align-content: center;
+      }
+      paper-item {
+        box-sizing: border-box;
+        width: 80%;
+      }
+      div {
+        width: 300px;
+      }
+    </style>
+
+    <paper-card id='card' heading='Time around the world'>
+      <div class="card-content">
+        <div class='card-flex'>
+          <template is="dom-repeat" items="{{timezones}}">
+            <paper-item-body two-line>
+              <div id="timer" inner-h-t-m-l=[[item.data]]></div>
```

Comment:
Why set innerhtml? You should be able to set up the `<span>` using data binding and use a `class` attribute for styling: `<span class="{{item.data.class}}">{{item.data.time}}</span>`